spec/lib/miq_automation_engine/miq_ae_service_spec.rb

```diff
@@ -189,6 +189,22 @@ module MiqAeServiceSpec
           result = miq_ae_service.create_notification!(:level => 'success', :audience => 'user', :message => 'test')
           expect(result).to be_kind_of(MiqAeMethodService::MiqAeServiceNotification)
         end
+
+        it "type of automate_tenant_info" do
+          allow(workspace).to receive(:persist_state_hash).and_return({})
```

Comment:
I don't think you need to use allow here.  I feel like you can expect these things to happen, right?